spt/OrangeBox/modules/ClientDLL.hpp

```diff
@@ -6,6 +6,7 @@
 #include <SPTLib\IHookableNameFilter.hpp>
 #include "..\spt-serverplugin.hpp"
 #include "..\..\SDK\igamemovement.h"
+#include "SDK/../../public/cdll_int.h"
```

Comment:
What is this include, why are you going into SDK, then up twice? Also please change `/` to `\`.